android/app/src/fdroid/play/listings/ru-RU/release-notes.txt

```diff
@@ -1,9 +1,8 @@
-• Сортировка меток и треков по имени
-• Данные OpenStreetMap по состоянию на 29 апреля
-• Если вы были залогинены в OSM, нужно перелогиниться
-• Сложные пешеходные тропы теперь отображаются коричневыми точками, очень сложные – чёрными редкими точками
-• Велодорожки теперь тёмно-синие, а где идут по пешеходным дорожкам – пунктирные
-• Под полупрозрачными мостами видны дороги
-• Исправлен импорт GPX из приложения Google Files
+• OpenStreetMap data as of May 28
+• search for US zip codes and UK (GB) postcodes
+• street name announcements in verbal TTS turn instructions (please enable it in settings), thanks to Will Bradley
+• export bookmarks and tracks in GPX file format
+• save bookmarks and tracks to a local device storage
+• add, edit, and display web links to menus of restaurants, bars, and other food services
```

Comment:
```suggestion
• Данные OpenStreetMap от 28 мая
• поиск по почтовым индексам в США и Великобритании
• объявления названий улиц в голосовых TTS-инструкциях при навигации (включается в настройках)
• экспорт меток и треков в формате GPX и сохранение их локально на устройстве
• добавление, редактирование и отображение веб-ссылок на меню ресторанов, баров и других заведений общественного питания
```

Reply:
@rtsisyk any help with translations into the languages you speak is appreciated.

Reply:
If I had enough time I would not ask for help. Thanks for your understanding and support.